docs/source/concepts/TheoreticalS.rst

```diff
@@ -0,0 +1,162 @@
+
+:math:`S^j(Q, \omega_{iklm}) = \frac{27Q^8}{9850}\left( TrB^j_{\omega_i} TrB^j_{\omega_k} TrB^j_{\omega_l}TrB^j_{\omega_m}  \right) exp\left(-Q^2 \beta^j coth^2\left(\frac{\hbar \omega_{iklm}}{2 k_B T}\right) \right)\sigma^j`
+
+In the similar way as for the second quantum order event one can reduce number of energy transitions by taking into account considered energy window and neglecting small S.
+
+After evaluating equations above one obtains discrete S for each quantum order and for each atom. In order to compare such a spectrum (:math:`S_{theory}`) one has to convolve it with experimental resolution:
+
+:math:`S_{theory}^{nj}(Q, \omega) = S_{discrete}^{nj}(Q, \omega) * f(\omega)`
+
+where:
+
+:math:`j` -- indicates j-th atoms
+
+:math:`n` -- indicates n-order event
+
+:math:`f(\omega)` -- resolution function
+
+For `Tosca <http://www.isis.stfc.ac.uk/instruments/tosca/tosca4715.html>`_  and Tosca-like instruments :math:`f(\omega)` has the following form:
+
+:math:`f(\omega)=1.0 / \sqrt{\sigma(\omega)  \pi}  \exp(-(\omega)^2  / \sigma(\omega))`
+
+where:
+
+:math:`\sigma(\omega) = A  \omega^2  + B  \omega + C`
+
+with A, B, C as constants.
+
+Moreover, in case of Tosca and Tosca-like instruments length of momentum transfer depends on frequency (*indirect geometry spectrometer*).
+The formula for Q is as follows:
+
+:math:`Q(\omega)=k2_i(\omega) + k2_f - 2  \sqrt{k2_i(\omega)  k2_f} angle`
+
+where:
+
+:math:`k2_i(\omega)=(\omega + E_{final})  \hbar/ (4  \pi)` expressed in the spectroscopic units
+
+:math:`k2_f=E_{final}  \hbar/(4 \pi)`
+
+with
+
+:math:`E_{final}` -- final energy on the crystal analyser in :math:`cm^{-1}`
+
+:math:`angle` -- cosines of crystal analyser angle in radians
+
+Current implementation
+++++++++++++++++++++++
+Calculation of theoretical S from ab-initio results is implemented in :ref:`Abins <algm-Abins>`. At the moment Abins supports
+`CASTEP <http://www.castep.org/>`_ and `CRYSTAL <http://www.crystal.unito.it/index.php>`_ DFT programs. As it comes to instruments,
+`Tosca <http://www.isis.stfc.ac.uk/instruments/tosca/tosca4715.html>`_ and Tosca-like instruments are supported.
+
+References
+++++++++++
+
+[1] W. Kohn et al., *Inhomogeneous Electron Gas*, Phys. Rev. B {\bf 136}, 864 (1964).
+
+[2] J. Howard, B.C. Boland, J. Tomkinson, *Intensities in inelastic neutron scattering spectra: a test of recent theory*, Chem. Phys. 77 (1983).
+
+[3] J. Howard and J. Tomkinson, *An analytical method for the calculation of the relative intensities of bending and stretching modes in inelastic neutron scattering spectra*, Chem. Phys. Letters 98 (1983).
+
+[4] P. C H Mitchell, S. F. Parker, A. J. Ramirez-Cuesta, J. Tomkinson, * Vibrational Spectroscopy with Neutrons With Applications in Chemistry, Biology, Materials Science and Catalysis *, ISBN: 978-981-256-013-1
+.. categories:: Concepts
```

Comment:
I figured out why it does not work. Add an extra line before categories, that way it will show up in the concepts page